content/tutorials/jwt-authentication.textile

```diff
@@ -273,43 +296,61 @@ The last part is to add the logic into this form. You'll instantiate Ably's Real
 ```[html]
 <script type="text/javascript">
 console.log('On login page now')
+
+
 function login(e) {
-    e.preventDefault();
-    console.log('Fetching JWT token from auth server')
-    const realtime = new Ably.Realtime({ authUrl: '/auth' });
-    realtime.connection.once('connected', function () {
-    console.log('Client connected to Ably using JWT')
-    alert("Client successfully connected to Ably using JWT auth")
-    })
-}
+  e.preventDefault();
+
+  const username = document.querySelector("#username").value;
+  const password = document.querySelector("#password").value;
+
+  if (!`${username}`.trim() || !`${password}`.trim()) {
+    return alert("Error! Require username and password");
+  }
+
+  // Send user details to the auth server
+  const authUrl = `/auth?clientId=${username}:${password}`;
+  const realtime = new Ably.Realtime({ authUrl });
+
+  console.log("Requesting Ably JWT token from auth server", username, password);
+
+  realtime.connection.once("connected", () => {
+    const { tokenDetails } = realtime.auth;
+    console.log("Success! You are connected with a JWT token", tokenDetails);
+    alert("Success! You are connected with a JWT token");
```

Comment:
I'm not sure why this text was changed - it now does not correspond to the original code and tutorial text, which was: `alert("Client successfully connected to Ably using JWT auth")` The new text doesn't really work because JWT is a JSON Web Token, so "Success! You are connected with a JWT token" would be "JSON Web Token token"!

I would stick with the original text as this is what is show in the tutorial text anyway:

![image](https://user-images.githubusercontent.com/25511700/183642400-3c70a3fb-670f-4fb4-a12e-d8af15c08ef7.png)
 
Note, same comment applies to line 319.

Reply:
corrected both instances